### ISSUE TITLE
Android - Missing Pause Buttons

### ISSUE THREAD

**Issue by 4oo4** [raicem/impulse-blocker#111]
I found the extension on Mozilla's list of Android compatible extensions, however the pause/unpause function on the block page seems to be missing. I can still unblock websites in settings but it would be nice to have this working.

![Screenshot_20240104-192239_Firefox](https://github.com/raicem/impulse-blocker/assets/15698835/f578da09-aafc-4691-935f-a89dfeeea73f)
